cumulusci/cumulusci.yml

```diff
@@ -140,6 +140,9 @@ tasks:
         class_path: cumulusci.tasks.datadictionary.GenerateDataDictionary
         options:
             release_prefix: $project_config.project__git__prefix_release
+    generate_and_load_from_yaml:
+        class_path: cumulusci.tasks.bulkdata.generate_and_load_data_from_yaml.GenerateAndLoadDataFromYaml
+        group: Data Operations
```

Comment:
Looks like the task docs need to be regenerated to include the new task

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/1659/commits/819742de10afde65cafa3401b2dc979a4f71a3af